packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+.rt-DataListLabel {
+  color: var(--gray-a11);
+  box-sizing: border-box;
+  display: block;
```

Comment:
- Sounds like we need color and highContrast props instead of hardcoding this
- Is `display: block` and `box-sizing` needed?

Reply:
@vladmoroz would you see `color` being set per label or at the Item or Root level?

Reply:
Definitely per label, you don't want the label and data to be always in sync

E.g. you might have to display an invalid or missing piece of data, and indicate that using a red label

Reply:
For sure, I more was asking if we want to accept `color` higher and only apply it to the `Label`. But keeping it connected with the affected component makes sense to me.